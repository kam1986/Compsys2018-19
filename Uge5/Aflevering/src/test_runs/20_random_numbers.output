ffffffffffffe6d9 ffffffffffffeab3 fffffffffffff18f fffffffffffff565 fffffffffffff68d fffffffffffff70e fffffffffffffb18 000000000000036c 00000000000003b7 000000000000076b 0000000000000ab2 0000000000000f33 0000000000000f7b 0000000000001290 00000000000014f3 0000000000001846 0000000000001d41 00000000000023a9 0000000000002706 
Branch predictions:      184   miss       71    missrate: 0.385870
Return predictions:       20   miss        0    missrate: 0.000000
L2-Cache reads:       20   writes:       19   miss:       39   missrate: 1.000000
I-Cache reads:      1041   writes:        0   miss:       20   missrate: 0.019212
D-Cache reads:       181   writes:      167   miss:       19   missrate: 0.054598
Execution finished at cycle 3234
