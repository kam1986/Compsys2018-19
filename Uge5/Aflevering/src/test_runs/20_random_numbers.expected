ffffffffffffe6d9 ffffffffffffeab3 fffffffffffff18f fffffffffffff565 fffffffffffff68d fffffffffffff70e fffffffffffffb18 000000000000036c 00000000000003b7 000000000000076b 0000000000000ab2 0000000000000f33 0000000000000f7b 0000000000001290 00000000000014f3 0000000000001846 0000000000001d41 00000000000023a9 0000000000002706 
